scripts/build/Platform/Linux/build_config.json

```diff
@@ -0,0 +1,36 @@
+{
+  "clean": {
+    "TAGS": [],
+    "COMMAND": "clean_linux.sh",
+    "PARAMETERS": {
+      "OUTPUT_DIRECTORY": "build",
+      "CMAKE_LY_PROJECTS": "AutomatedTesting"
```

Comment:
Does the project matter here at all? My only concern would be old MPS codegen artifacts not getting flushed.

Reply:
Not really, it was just the default config in the o3de AR. AR will still use `AutomatedTesting` if it's not defined. The codegen artifacts will get flushed once we get the Jenkinsfile changes in, since it will start with a clean build.